addons/google_account/models/google_service.py

```diff
@@ -193,10 +193,6 @@ def _do_request(self, uri, params=None, headers=None, method='POST', preuri="htt
             except:
                 pass
         except requests.HTTPError as error:
-            if error.response.status_code in (204, 404):
-                status = error.response.status_code
-                response = ""
-            else:
-                _logger.exception("Bad google request : %s !", error.response.content)
-                raise error
+            _logger.exception("Bad google request : %s !", error.response.content)
+            raise error
```

Comment:
Are you sure this change is only applicable for your usecase and no other?
You are making modifications in 14.0. I am not ready to risk breaking everything for a simple error....